build-aux/snap/snapcraft.yaml

```diff
@@ -128,3 +129,30 @@ parts:
       cp -a fc-cache-bionic $SNAPCRAFT_PART_INSTALL/bin/fc-cache-v7
     prime:
       - bin/fc-cache-v7
+  apparmor:
+    plugin: autotools
+    build-packages: [ bison, flex, gettext ]
+    source: https://launchpad.net/apparmor/3.0/3.0.3/+download/apparmor-3.0.3.tar.gz
```

Comment:
I would rather be explicit about it since it is not impossible that a new AppArmor might introduce changes that could have subtle effects on policy etc.

Reply:
I think I'm fine with explicit, but we need a policy about who/how we do the tracking

Reply:
Given the security team has a very close relationship to apparmor upstream then I think it makes most sense that we would do the tracking and updating here so I am happy to take this on myself.